openupgrade_scripts/scripts/hr_holidays/14.0.1.5/upgrade_analysis_work.txt

```diff
@@ -37,7 +41,11 @@ hr_holidays  / hr.leave.allocation      / interval_number (integer)     : now a
 hr_holidays  / hr.leave.allocation      / interval_unit (selection)     : now a function
 hr_holidays  / hr.leave.allocation      / manager_id (many2one)         : now a function
 hr_holidays  / hr.leave.allocation      / mode_company_id (many2one)    : now a function
+# NOTHING TO DO
+
 hr_holidays  / hr.leave.allocation      / name (char)                   : not stored anymore
+# DONE: rename field to private_name
```

Comment:
Put together both lines and use the text: `DONE: pre-migration: rename name > private_name`

Reply:
Done

Reply:
This is not what I said. Move this line to be together with the previous change and only one comment line for both:
```
hr_holidays  / hr.leave                 / private_name (char)           : NEW
hr_holidays  / hr.leave                 / name (char)                   : not stored anymore
# DONE: pre-migration: rename name > private_name
```
(and the same for the other model)

Reply:
I understand, done